nutkit/frontend/driver.py

```diff
@@ -67,6 +67,28 @@ def send_and_receive(self, req, timeout=None, hooks=None, *,
         return self.receive(timeout=timeout, hooks=hooks,
                             allow_resolution=allow_resolution)
 
+    def execute_query(self, cypher, params=None, config=None,
+                      routing=None, database=None,
+                      impersonated_user=None, bookmark_manager=...):
+        config = {}
+        if (routing):
+            config["routing"] = routing
+        if (database):
+            config["database"] = database
+        if (impersonated_user):
+            config["impersonatedUser"] = impersonated_user
```

Comment:
```suggestion
        if routing:
            config["routing"] = routing
        if database:
            config["database"] = database
        if impersonated_user:
            config["impersonatedUser"] = impersonated_user
```
This isn't JavaScript :wink: 

But why do we even omit them? `SessionRun` e.g., always transmits `params`.

Reply:
So what about 
```python
        config = {
            "routing": routing,
            "database": database,
            "impersonatedUser": impersonated_user,
        }
```
instead?

Reply:
This is pythonscript.